.pre-commit-config.yaml

```diff
@@ -113,6 +100,8 @@ repos:
           - /*| *| */
       - id: insert-license
         files: (\.yaml$|\.yml$)
+        # exclude helm chart templates
+        exclude: ^projects/control-service/projects/helm_charts
```

Comment:
why excluding the copyright notices from yaml files? 
having those, is in accordance with company legal

Reply:
Because helm can't parse the yaml files for some reason if they have comments on the first line.

Reply:
I did helm lint, and the chart seems well formatted. Also, helm documentation states that yaml templates do support comments

Reply:
Actual error we're getting

```
Error: UPGRADE FAILED: error validating "": error validating data: apiVersion not set
```

https://gitlab.com/vmware-analytics/versatile-data-kit/-/jobs/3813924321

Reply:
should we remove the copyright notice, or is there any other way to keep it? removing it seems harshly and against legal

Reply:
I can check if we can add it to the bottom of the file instead.

Reply:
> Put the two copyright notices ("Copyright Notice" and "Copying Permission Statement") together, near the top of each source file.

Reply:
> Put the two copyright notices ("Copyright Notice" and "Copying Permission Statement") together, near the top of each source file.
Bummer 😕

I'll dig a bit deeper, then. Maybe repro locally. But this is most likely what's causing the problem. Maybe we need to update helm in the runner image if it's been fixed upstream.

Reply:
You can remove the copyright notice. There were not there before as well (until very recently). And that was on purpose, not an omission. When we open-sourced the project a year ago I consulted and it was ok to remove it if it would break the build and there was no alternative.

Reply:
To explain a bit more:  Under the Apache License (2.0) it is recommended that all source files include a copyright notice to protect intellectual property and provide clarity around ownership of the code. However, the Apache License does not require a copyright notice to be included in the source files.

The Apache License has specific requirements for attribution and notices that must be included in the LICENSE and NOTICE files distributed with the project (which we have), but it does not require that copyright notices be included in individual source files. That being said, having a copyright notice in each source file can help prevent confusion about the origins of the code and can help protect against unauthorized use or distribution. Hence we put it everywhere

Reply:
I actually found a way to keep the licenses, although it might result in some annoying debug output. Also turned out that I didn't remove the correct license headers. I'll update the PR shortly.